llvm/lib/Target/AArch64/AArch64Subtarget.cpp

```diff
@@ -491,13 +491,13 @@ bool AArch64Subtarget::isStreamingCompatible() const {
 }
 
 bool AArch64Subtarget::isNeonAvailable() const {
-  return hasNEON() && !isStreaming() && !isStreamingCompatible();
+  return (hasNEON() || hasSMEFA64()) && !isStreaming() &&
```

Comment:
There are no tests for the changes in this file. These changes are really the reason for adding this feature, and it's important to test it.

Reply:
I've added the file [sve-streaming-mode-fixed-length-fp-reduce-fa64.ll](https://github.com/llvm/llvm-project/pull/70809/commits/9d2078b903a9ae8d54cdf5e34b4d88a10c0d372e#diff-cdcb53699f7848dacb5e12e4c40f2418be9ef30428b5308016f91b3bfe224d1a)